source/_components/sensor.transport_nsw.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Transport NSW"
+description: "Instructions on how to integrate timetable data for Transport NSW (Australia) within Home Assistant."
+date: 2018-08-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.76
```

Comment:
This release has been shipped quite some time ago.
Please update this to the next major release.